### PR TITLE
feat: add information on multiple VPCs.

### LINKED ISSUE #3560: [Docs RFC] Clarify language for VPC Peering
Clarify the language around multi-attach.

### PR DESCRIPTION
**Pull request by billy-the-fish**

### CODE REVIEW

File: use-timescale/vpc.md [hunk 1]
```diff
@@ -11,16 +11,14 @@ cloud_ui:
 
 # Secure your $CLOUD_LONG services with VPC Peering and AWS PrivateLink
 
-You use Virtual Private Cloud ($VPC) peering to ensure that your $CLOUD_LONG services are 
+You use Virtual Private Cloud ($VPC) peering to ensure that your $SERVICE_LONG are
```
Comment (atovpeko) [inline use-timescale/vpc.md]:
```suggestion
You use Virtual Private Cloud ($VPC) peering to ensure that your $SERVICE_LONGs are 
```

File: use-timescale/vpc.md [hunk 2]
```diff
@@ -35,9 +33,19 @@ peering connection to your Customer VPC, you use AWS Security Groups to
...
+* You can attach:
+  * Any number of Customer $VPCs to a $CLOUD_LONG $VPC.
+  * A $SERVICE_LONG to a single $CLOUD_LONG $VPC at a time.
+  
+      The $SERVICE_SHORT and $VPC must be in the same AWS region. To support multiple AWS regions, you need multiple 
+      $CLOUD_LONG $VPCs.
+  * Multiple $SERVICE_LONGs to the same $CLOUD_LONG $VPC.
+* You cannot attach a $SERVICE_LONG to multiple $CLOUD_LONG $VPCs at the same time.
+
+The number of $CLOUD_LONG $VPCs you have in your project depends on your [pricing plan][pricing-plans].
```
Comment (atovpeko) [inline use-timescale/vpc.md]:
```suggestion
The number of $CLOUD_LONG $VPCs you can create in your project depends on your [pricing plan][pricing-plans]. 
```

File: use-timescale/vpc.md [hunk 3]
```diff
@@ -69,9 +77,21 @@ between $CLOUD_LONG and your own VPC in a logically isolated virtual network.
 
 1.  In [$CONSOLE > VPC][console-vpc], click `New VPC`.
 
-    The number of VPCs you can attach to your project depends on your [pricing plan][pricing-plans]. If you
-    need more VPCs either contact contact [support@timescale.com](mailto:support@timescale.com) or change 
-    your pricing plan in $CONSOLE. Each $CLOUD_LONG VPC can have as many peering connections as you need.
+    ![$CLOUD_LONG new $VPC](https://assets.timescale.com/docs/images/console-add-vpc.png)
```
Comment (atovpeko) [inline use-timescale/vpc.md:80]: Don't think variables will work in the alt tag

File: use-timescale/vpc.md [hunk 4]
```diff
@@ -69,9 +77,21 @@ between $CLOUD_LONG and your own VPC in a logically isolated virtual network.
...
+    * You can attach:
+      * Any number of Customer $VPCs to a $CLOUD_LONG $VPC.
+      * A $SERVICE_LONG to a single $CLOUD_LONG $VPC at a time.
+
+        The $SERVICE_SHORT and $VPC must be in the same AWS region. To support multiple AWS regions, you need multiple
+        $CLOUD_LONG $VPCs.
+      * Multiple $SERVICE_LONGs to the same $CLOUD_LONG $VPC.
+      * You cannot attach a $SERVICE_LONG to multiple $CLOUD_LONG $VPCs at the same time.
+
+    The number of $CLOUD_LONG $VPCs you have in your project depends on your [pricing plan][pricing-plans].
```
Comment (atovpeko) [inline use-timescale/vpc.md]:
```suggestion
    The number of $CLOUD_LONG $VPCs you can create in your project depends on your [pricing plan][pricing-plans].
```

File: use-timescale/vpc.md (hunk 3 above)
Comment (billy-the-fish) [reply use-timescale/vpc.md:80]: They do :-).

**Review by Khyme — APPROVED** [review]

File: use-timescale/vpc.md [hunk 5]
```diff
@@ -11,33 +11,41 @@ cloud_ui:
...
-The number of VPCs you can attach to your project depends on your [pricing plan][pricing-plans]. If you 
-need more VPCs either contact contact [support@timescale.com](mailto:support@timescale.com) or change your
-pricing plan in $CONSOLE. Each $CLOUD_LONG VPC can have as many peering connections as you need.
+peering connection to your Customer $VPC, you use AWS Security Groups to 
+restrict the services in your Customer $VPC that are visible to the Peering $VPC.
+The last step is to attach individual services to the Peering $VPC. 
+
+* You define each $CLOUD_LONG $VPC on a [$CLOUD_LONG project level][project-members]. 
+* You can attach:
+  * Any number of Customer $VPCs to a $CLOUD_LONG $VPC.
```
Comment (miqrc) [inline use-timescale/vpc.md]: The max is 50 (AWS max number of VPC peerings)

File: use-timescale/vpc.md [hunk 6]
```diff
@@ -11,33 +11,41 @@ cloud_ui:
...
+  * A $SERVICE_LONG to a single $CLOUD_LONG $VPC at a time.
+  
+      The $SERVICE_SHORT and $VPC must be in the same AWS region. To support multiple AWS regions, you need multiple
```
Comment (miqrc) [inline use-timescale/vpc.md]: This is not 100% true: Even if the customer creates more Timescale VPCs in different regions, the service still is in a given region, and they won't be able to attach it to a VPC in a different region.
I would change it to: The service and Timescale VPC must be in the same AWS region, but you can peer a Customer VPC and Timescale VPC that are in different regions.  
(Please, rephrase as desired).

File: use-timescale/vpc.md [hunk 7]
```diff
@@ -69,9 +77,21 @@ between $CLOUD_LONG and your own VPC in a logically isolated virtual network.
 
 1.  In [$CONSOLE > VPC][console-vpc], click `New VPC`.
 
-    The number of VPCs you can attach to your project depends on your [pricing plan][pricing-plans]. If you
-    need more VPCs either contact contact [support@timescale.com](mailto:support@timescale.com) or change 
-    your pricing plan in $CONSOLE. Each $CLOUD_LONG VPC can have as many peering connections as you need.
+    ![$CLOUD_LONG new $VPC](https://assets.timescale.com/docs/images/console-add-vpc.png)
+
+    * You can attach:
+      * Any number of Customer $VPCs to a $CLOUD_LONG $VPC.
```
Comment (miqrc) [inline use-timescale/vpc.md]: same as above

File: use-timescale/vpc.md [hunk 8]
```diff
@@ -69,9 +77,21 @@ between $CLOUD_LONG and your own VPC in a logically isolated virtual network.
...
+      * A $SERVICE_LONG to a single $CLOUD_LONG $VPC at a time.
+
+        The $SERVICE_SHORT and $VPC must be in the same AWS region. To support multiple AWS regions, you need multiple
```
Comment (miqrc) [inline use-timescale/vpc.md]: same as above

**Review by miqrc — CHANGES_REQUESTED** [review]
Some changes please

**Review by miqrc — APPROVED** [review]

**Pull request merged**